workflow-service/generated/openapi/openapi.json

```diff
@@ -259,6 +259,40 @@
       }
     },
     "/api/v1/workflows" : {
+      "get" : {
+        "operationId" : "getStatusByProjectId",
+        "parameters" : [ {
+          "in" : "query",
+          "name" : "projectId",
+          "required" : false,
```

Comment:
that should have rendered as `required`

Reply:
I changed it to false in the controller, so that we can return all executed workflows for this user